PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java

```diff
@@ -75,87 +77,106 @@
     protected JLabel workerNumberOfContainersLabel;
     protected JTextField errorMessageField;
     protected JPanel provisionDialogPanel;
-    protected JButton refreshButton;
+    protected JButton refreshButton = new JButton(new ImageIcon(CommonConst.RefreshIConPath));
     protected JLabel storageRootPathLabel;
     protected JComboBox sparkVersionComboBox;
     protected JLabel sparkVersionLabel;
 
+    @NotNull
+    private final List<TextWithErrorHintedField> allTextFields = Arrays.asList(clusterNameField, sparkEventsField);
```

Comment:
Why not to define these two fields as `Stream`? I saw all references are converting them to stream.